### PR TITLE
Custom url

### PR DESCRIPTION
**Pull request by billysutomo**
Hi, this is to implement custom URL flag
But if not supplied, it will use the default value "proxy"

### CODE REVIEW

File: huproxy.go [hunk 1]
```diff
@@ -33,6 +34,7 @@ var (
 	dialTimeout      = flag.Duration("dial_timeout", 10*time.Second, "Dial timeout.")
 	handshakeTimeout = flag.Duration("handshake_timeout", 10*time.Second, "Handshake timeout.")
 	writeTimeout     = flag.Duration("write_timeout", 10*time.Second, "Write timeout.")
+	url              = flag.String("url", "proxy", "Address to listen to.")
```
Comment (ThomasHabets) [inline huproxy.go]: Address -> Path

File: huproxy.go [hunk 2]
```diff
@@ -109,7 +111,8 @@ func main() {
 
 	log.Printf("huproxy %s", huproxy.Version)
 	m := mux.NewRouter()
-	m.HandleFunc("/proxy/{host}/{port}", handleProxy)
+	fullUrl := fmt.Sprintf("/%s/{host}/{port}", *url)
+	m.HandleFunc(fullUrl, handleProxy)
```
Comment (ThomasHabets) [inline huproxy.go]: Skip the short term variable, just inline.

**Review by ThomasHabets — CHANGES_REQUESTED** [review]

File: huproxy.go (hunk 1 above)
Comment (billysutomo) [reply huproxy.go]: Thanks for reviewing. Updated as requested.

File: huproxy.go (hunk 2 above)
Comment (billysutomo) [reply huproxy.go]: Thanks for reviewing. Updated as requested.

**Comment by ThomasHabets** [comment]
Thanks!

**Pull request merged**